src/Console/Util/PhpProcess.php

```diff
@@ -63,7 +63,23 @@ public function start(callable $callback = null, array $env = null): void
         $phpConfig = new PhpConfig();
 
         $phpConfig->useOriginal();
+
+        // As of 1.3.2 xdebug-handler won't update $_ENV if it is in use.
```

Comment:
Is it about these lines of code?

https://github.com/composer/xdebug-handler/blob/0af624bc9c20a3c9252793625bbe976af082d401/src/Process.php#L145-L159